docs/VnVPlan/VnVPlan.tex

```diff
@@ -74,8 +75,8 @@ \section{Revision History}
 
 \listoftables
 
-\listoffigures
-\wss{Remove this section if it isn't needed}
+% \listoffigures
+% \wss{Remove this section if it isn't needed}
```

Comment:
The rubric says:
> Has all required elements: Title page, incl. names, Table of Contents. List of Figures, List of Tables. All pages are numbered. Every figure has a caption and every table has a heading. Revision History. Supporting Material –appendices, etc. Sections for General Information, Plan, System Tests, Unit Tests; document is easy to navigate

So idk if we should remove it even if the comments say we can.